examples/arithmetic_main.lua

```diff
@@ -0,0 +1,4 @@
+arithmetic = require "examples.arithmetic"
```

Comment:
Each example could be inside its own dir. For instance
```
examples/arithmetic/arithmetic.pln
examples/arithmetic/main.lua
```
And so on.

Reply:
Okay, done.